force-app/main/default/lwc/lwcChatbotQuery/lwcChatbotQuery.js

```diff
@@ -22,22 +22,23 @@ export default class LwcChatbotQuery extends LightningElement {
   }
 
   async refreshToken() {
-    window.sessionStorage.removeItem(SESSIONSTORAGE_TOKEN_KEY);
+    window.sessionStorage.removeItem(SESSIONSTORAGE_ENDPOINT_KEY);
     await this.setupEndpoint();
   }
 
   /**
    * Setup a search endpoint. This means querying an Apex method to get endpoint data that includes a search token.
    */
   async setupEndpoint() {
-    const sessionStorageToken = window.sessionStorage.getItem(SESSIONSTORAGE_TOKEN_KEY);
-    if (sessionStorageToken && sessionStorageToken !== 'undefined') {
-      this.endpoint = new SearchEndpoint(sessionStorageToken, DEFAULT_PLATFORM_ENDPOINT, DEFAULT_PLATFORM_ENDPOINT);
+    const sessionStorageEndpoint = window.sessionStorage.getItem(SESSIONSTORAGE_ENDPOINT_KEY);
+    if (sessionStorageEndpoint && sessionStorageEndpoint !== 'undefined') {
+      const data = JSON.parse(sessionStorageEndpoint);
+      this.endpoint = new SearchEndpoint(data.token, data.clientUri || data.platformUri, data.clientUri || data.analyticUri);

```

Comment:
Did you make sure to test the analyticUri as well you might need to add `/rest/ua` on that.